app/views/home/ordergroup.html.haml

```diff
@@ -9,6 +9,11 @@
       %p
         %b= heading_helper(Ordergroup, :available_funds) + ':'
         = number_to_currency(@ordergroup.get_available_funds())
+      - if FinancialTransactionClass.count > 1
```

Comment:
What about introducing a method on `FinancialTransactionClass` that tells whether the financial transactions class feature is needed?